### ISSUE TITLE
Make annotations clickable

### ISSUE THREAD

**Issue by Treutler** [Treutler/MetFamily#25]
The annotation legend lists all displayed annotations.
Make annotations clickable to select all precursors with this annotation (just evaluate y position of the click).
Could be equivalent to a search, but where to reset this selection?.

**Comment by Treutler** [comment]
Consider doingthis in the legends plot as well as in the annotations tab.

**Comment by sneumann** [comment]
Good idea, especially in the presence of many annotations. Will be considered during ggplotly rewrite
